apps/docs/content/guides/platform/access-control.mdx

```diff
@@ -1,85 +1,258 @@
 ---
 title: 'Access Control'
-description: 'Roles and permissions at the organization level'
+description: 'Roles and permissions at the organization and project levels'
 ---
 
-Supabase provides granular access controls to manage permissions across your organizations.
-For each organization, a member can have one of the following roles:
+Supabase provides granular access controls to manage permissions across your organizations and projects.
 
-- Owner
-- Administrator
-- Developer
+For each organization and project, a member can have one of the following roles:
+
+- **Owner**: full access to everything in organization and project resources.
+- **Administrator**: full access to everything in organization and project resources except updating organization settings, transferring projects outside of the organization, and adding new owners.
```

Comment:
Admins can't add owners - but they can add new members

```suggestion
- **Administrator**: full access to everything in organization and project resources except updating organization settings, transferring projects outside of the organization, and adding new members.
```

Reply:
@joshenlim that's what is conveyed here - _except_ updating org settings, ... adding new owners. Feel free to change the structure if thats not clear.

Reply:
ohhhhh okay gotcha on this!